.devcontainer/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apt-get update \
     && apt-get install -qq -y --no-install-recommends \
     git \
     python3-pip \
+    ros-humble-controller-manager-msgs \
```

Comment:
This shouldn't be needed and should be avoided. If the dependency is correctly defined in `package.xml` it should be installed automatically. This would duplicate where we define dependencies which should be avoided @Cyano0

Reply:
the correct way to install dependencies is via `rosdep install -i --from-paths src/` which is also automatically run when the container is created.